snuba/datasets/__init__.py

```diff
@@ -108,6 +109,9 @@ def get_timestamp_column(self) -> str:
         """
         return NotImplementedError('dataset does not support get_timestamp_column')
 
+    def get_rate_limits(self, request):
```

Comment:
Need to do this for other datasets.

The `__init__` will only get `[GlobalRateLimit(request)]`.